specification/network/resource-manager/Microsoft.Network/stable/2024-03-01/examples/WafListAllPolicies.json

```diff
@@ -148,6 +148,69 @@
                     "ruleSetType": "OWASP",
                     "ruleSetVersion": "3.2"
                   }
+                ],
+                "allowlists": [
+                  {
+                    "matchVariable": "RequestURI",
+                    "valueMatchOperator": "Contains",
+                    "values": [
+                      "path1",
+                      "path2",
+                      "path3"
+                    ],
+                    "allowlistManagedRuleSets": [
+                      {
+                        "ruleSetType": "OWASP",
+                        "ruleSetVersion": "3.2"
+                      }
+                    ]
+                  },
+                  {
+                    "matchVariable": "RequestHeader",
+                    "selectorMatchOperator": "StartsWith",
+                    "selector": "test",
+                    "valueMatchOperator": "Contains",
+                    "values": [
+                      "val1",
```

Comment:
Will be better to use real illustrative header values here...

Reply:
Updated to use real values